flexx/_config.py

```diff
@@ -9,4 +9,6 @@
                          'it is closed.'),
     ssl_certfile=('', str, 'cert file for https server.'),
     ssl_keyfile=('', str, 'key file for https server.'),
+    hosts_whitelist=('', str, 'Comma separated list of allowed <host>:<port> '
+                              'values to pass cross-origin checks.'),
```

Comment:
Nitpicking, but I'd prefer `host_whitelist`, because "list" already implied plural, and googling it seems to suggests a few projects also using that name.